MC-2715 Client filtering preview (#322)
hazelcast/management-center-docs: docs/modules/clusters/pages/client-filtering.adoc

Shows the result of applying filter lists on currently connected clients in real time. Can be hidden by a toggle control (*visible by default*).

image:ROOT:ClientFilteringPreview.png[Client Filtering Preview]

*Allowed clients* - clients that remain connected.

*Denied clients* - clients that disconnect.

.Default table view
[cols="30%s,70%a"]
|===
|Column name|Description

|Name
|Client name.
|UUID
|Client unique identifier.
|IP Address
|Client IP address without hostname.
|Labels
|List of client labels.

|===

.Extra columns (can be used in custom views)
[cols="30%s,70%a"]
|===
|Column name|Description

|Address
|Client address.
|Host Name
|Canonical client hostname.
|Client Type
|Type of client. This field typically denotes the client’s programming language and indicates whether it is an Enterprise Java client. Although non-Java clients can also connect to Enterprise clusters, the Java client includes Enterprise features that non-Java clients do not.
|Client Version
|Version of the client codebase.

|===

[[cleint-filtering-deploy-confirmation]]
== Deploy confirmation

image:ROOT:ClientFilteringDeployConfirmation.png[Client Filtering Deploy Comfirmation]

> The deployment of filter lists must be confirmed by clicking the *Confirm Deploy* button in a modal.

Reviewer: ```suggestion
The deployment of filter lists must be confirmed by clicking the *Confirm Deploy* button.
```

> This modal provides a summary of clients connectivity after the current filter lists have been applied.

Reviewer: ```suggestion
A summary of client connectivity is shown after the current filter lists have been applied.
```

<<client-filtering-preview, Preview>> can be enabled by clicking a toggle control.

> IMPORTANT: If the current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) will be shown on the top of a modal.

Reviewer: ```suggestion
IMPORTANT: If the current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) is displayed on the top of a modal.
```

> IMPORTANT: If current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) will be shown on the top of a modal.

Reviewer: ```suggestion
IMPORTANT: If the current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) will be shown on the top of a modal.
```
